### PR TITLE
feat(RPS-234): Jogging panel layout update

### PR DESCRIPTION
**Pull request by vgerber**

### CODE REVIEW

File: src/components/jogging/JoggingCartesianAxisControl.tsx [hunk 1]
```diff
@@ -35,54 +38,81 @@ export const JoggingCartesianAxisControl = externalizeComponent(
...
+        if (disabled) {
+          return
+        }
+        setBorderColor(colors?.buttonBackgroundColor?.pressed)
+        if (ev.button === 0) {
+          startJogging(direction)
+        }
+      }
+
+      function onPointerUp() {
```
Comment (evrys) [inline src/components/jogging/JoggingCartesianAxisControl.tsx:88]: I'd probably rename this to onPointerUpOrOut and put the stopJogging call inside it

File: src/index.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ export * from "./components/jogging/JoggingCartesianAxisControl"
 export * from "./components/jogging/JoggingJointRotationControl"
 export * from "./components/jogging/JoggingPanel"
 export type { JoggingStore } from "./components/jogging/JoggingStore"
+export * from "./components/jogging/PoseCartesianValues"
```
Comment (evrys) [inline src/index.ts:7]: Hmm, what about the joint values?

**Review by evrys — COMMENTED** [review]
Mostly looks good code wise, just two little comments 😄 

Would also be good to give the velocity slider label a fixed min width so the slider doesn't change width while you're using it

File: src/components/jogging/JoggingCartesianAxisControl.tsx (hunk 1 above)
Comment (vgerber) [reply src/components/jogging/JoggingCartesianAxisControl.tsx]: Done

File: src/index.ts (hunk 2 above)
Comment (vgerber) [reply src/index.ts:7]: Added PoseJointValues as well

**Pull request merged**